src/components/molecules/TableComponent/TableComponent.scss

```diff
@@ -0,0 +1,99 @@
+@use "scss/attendee/mixins.scss"as *;
```

Comment:
This file (`TableComponent`) should be a SCSS module. Take a look at how some of the other code is written.